src/components/course/SortParticipant.tsx

```diff
@@ -1,72 +1,122 @@
+    };
+
     const menu = (
-        <Menu>
-          <Menu.Item>
-            <a target="_blank"   onClick={() => sort('firstname')}>
+      <Menu>
+        <Menu.Item>
+          <a target="_blank" onClick={() => sort('firstname')}>
             Vorname
-            </a>
-          </Menu.Item>
-          <Menu.Item>
-            <a target="_blank"  onClick={() => sort('lastname')}>
+          </a>
+        </Menu.Item>
+        <Menu.Item>
+          <a target="_blank" onClick={() => sort('lastname')}>
             Nachname
-            </a>
-          </Menu.Item>
-          <Menu.Item>
-            <a target="_blank" onClick={() => sort('grade')}>
-              Note
-            </a>
-          </Menu.Item>
-        </Menu>
-      );
+          </a>
+        </Menu.Item>
+        <Menu.Item>
+          <a target="_blank" onClick={() => sort('grade')}>
+            Note
+          </a>
+        </Menu.Item>
+      </Menu>
+    );
 
-  return (
-    <Dropdown overlay={menu} placement="bottomLeft" arrow>
-    <Button style={{width: '100%'}}>Teilnehmer sortieren <SortAscendingOutlined /></Button>
-  </Dropdown>
-  );
-});
+    return (
+      //   <Dropdown overlay={menu} placement="bottomLeft" arrow>
+      //   <Button style={{width: '100%'}}>Teilnehmer sortieren <SortAscendingOutlined /></Button>
+      // </Dropdown>
+      <>
+        <div className={classes.sortParticipant}>
+          <ul>
+            <li>
+              <p style={{ paddingTop: '5px' }}>Sortieren:</p>
+            </li>
+            <li>
+              <AntdButton onClick={() => sort('firstname')}>
+                Vorname
+                <Icons.SortSolid
+                  style={{ marginLeft: '5px', height: '12px' }}
+                />
+              </AntdButton>
+            </li>
+            <li>
+              <AntdButton onClick={() => sort('lastname')}>
+                Nachname
+                <Icons.SortSolid
+                  style={{ marginLeft: '5px', height: '12px' }}
+                />
+              </AntdButton>
+            </li>
+            <li>
+              <AntdButton onClick={() => sort('email')}>
+                E-Mail-Addresse
+                <Icons.SortSolid
+                  style={{ marginLeft: '5px', height: '12px' }}
+                />
+              </AntdButton>
+            </li>
+            <li>
+              <AntdButton onClick={() => sort('grade')}>
+                Note
+                <Icons.SortSolid
+                  style={{ marginLeft: '5px', height: '12px' }}
+                />
+              </AntdButton>
+              <li>
+                <AntdButton onClick={() => sort('schooltype')}>
+                  Schultyp
+                  <Icons.SortSolid
+                    style={{ marginLeft: '5px', height: '12px' }}
+                  />
+                </AntdButton>
+              </li>
+            </li>
+          </ul>
+        </div>
+      </>
+    );
+  }
+);
 
```

Comment:
There are again many linter errors in this file. Please make sure you have merged `master` into this branch.